precompiles/assets-erc20/ERC20.sol

```diff
@@ -109,3 +109,33 @@
         uint256 value
     );
 }
+
+    /**
+     * @title Extension for ERC20 interface
+     * @dev Extended functions with minimum balance check as well as mint & burn.
+     */
+    interface IERC20Plus is IERC20 {
+
+    /**
+     * @dev Returns minimum balance an account must have to exist
+     * Selector: b9d1d49b
+     */
+    function minimumBalance() external view returns (uint256);
+
+    /**
+     * @dev Mints the specified amount of asset for the beneficiary.
+     * This operation will increase the total supply.
+     * Only usable by asset admin.
+     * Selector: 40c10f19
+     */
+    function mint(address beneficiary, uint256 amount) external returns (bool);
```

Comment:
in the OpenZeppelin's [reference](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol) implementation both `mint` and `burn` are `internal` function, therefore can only be called within the smart contract and derived contracts

Reply:
OpenZeppelin isn't referenced for `IERC20Plus`, only for the original.

Reply:
And since they are internal, they are named `_mint` and `_burn`

Reply:
Ok, but can you please explain what you're trying to say?

Because I don't see any impediments for merge.